src/app/components/book-tile/book-tile.js

```diff
@@ -117,11 +118,14 @@ function useBookState(id) {
 
 export default function BookTile({ book: [book] }) {
     const { coverUrl, title, slug } = book;
-    const state = useBookState(book.id);
-    const classes = cn({'book-tile': true, 'coming-soon': state === 'coming_soon'});
+    const info = useBookInfo(book.id);
+    const comingSoon = info?.state === 'coming_soon';
+    const assignable = info?.assignable_book === true;
+    const classes = cn({'book-tile': true, 'coming-soon': comingSoon, assignable});
 
     return (
         <div className={classes}>
+            {assignable && <AssignableBadge />}
```

Comment:
Should we make the badge clickable too? When I was testing it, it seemed like the entire cover area should be clickable (though may have to wrap in a separate `a` tag so the tooltip hover still works.)

Reply:
It works.